flow/table.go

```diff
@@ -107,6 +110,11 @@ type Operation struct {
 
 // NewTable creates a new flow table
 func NewTable(updateHandler *Handler, expireHandler *Handler, nodeTID string, opts ...TableOpts) *Table {
+	appTimeout := make(map[string]time.Duration)
+	for key := range config.GetConfig().GetStringMap("flow.table.app_timeout") {
+		appTimeout[strings.ToLower(key)] = time.Second * time.Duration(config.GetConfig().GetFloat64("flow.table.app_timeout." + key))
```

Comment:
I would suggest to use upper case as it is like this for all the application and then no need to lower flow.Application in the update loop

Reply:
Please provide an example in the default configuration file: etc/skydive.yml.default